package.json

```diff
@@ -4,12 +4,15 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "nyc --reporter=lcov mocha ./test/index.js --require babel-register --exit"
```

Comment:
Have a separate script for generating coverage report and another script for running your tests.
Something along "test:coverage": "...." and "test": "..."

Reply:
Thanks for your feedback. I've removed the coverage line from the test script.